ng-package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "./node_modules/ng-packagr/ng-package.schema.json",
+  "deleteDestPath": false,
+  "lib": {
+    "entryFile": "./src/index.ts",
+    "whitelistedNonPeerDependencies": ["."],
+    "umdModuleIds": {
+      "tslib": "tslib",
+      "lodash": "_"
```

Comment:
you might need to add `vanilla-tilt: vanillaTilt` in here (see warnings when you build)